Fix route_guide ruby example

kpayson64 opened the pull request:
Note I skipped the ```&``` because it suggests using 2 terminals.

murgatroid99 commented on docs/tutorials/basic/ruby.md:
````diff
@@ -294,11 +294,11 @@ Run the server, which will listen on port 50051:
 
 ```
 $ # from examples/ruby
-$ bundle exec route_guide/route_guide_client.rb ../node/route_guide/route_guide_db.json &
````
This shouldn't be removed. The client uses it too.

kpayson64 commented:
Addressed comments.
Also forgot to mention that the driving factor for this change is that route_guide_db.json moved in the node folder.  Do we want to just place a copy in the ruby example directory?

murgatroid99 commented:
I would be fine with doing it either way.

kpayson64 commented:
Lets leave it as is for now then.  I've addressed the comments.
